media-libs/libgfx/libgfx-1.1.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Simplify the creation of computer graphics software"
+HOMEPAGE="http://mgarland.org/software/libgfx.html"
+
+if [[ ${PV} == "1.1.1" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mjgarland/libgfx"
+	EGIT_BRANCH="branch-1.1.1"
```

Comment:
The difficulty with `git-r3.eclass` is that it skips the regular fetching logic, and therefore skips the Manifest file. Because it skips the Manifest check we can't have KEYWORDS in it for QA reasons (and pkgcheck will complain about it).

However, we can use GitHub's ability of creating tarballs for any commit as a workaround:
```
COMMIT="f28ca5f0df3044f6399d6214faf66daa36712b91"
SRC_URI="https://github.com/mjgarland/libgfx/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
S="${WORKDIR}/${PN}-${COMMIT}"
```
Because now we don't use `git-r3.eclass` and have the tarball in plain old `SRC_URI` instead we can have KEYWORDS.

Reply:
Thank you for your review. Should I drop version 1.1.0? When I run `pkgcheck scan .` I get a QA message:
`media-libs/libgfx`
`RedundantVersion: version 1.1.0: slot(0) keywords are overshadowed by version: 1.1.1`

Reply:
>  Should I drop version 1.1.0? When I run `pkgcheck scan .` I get a QA message:
> `media-libs/libgfx`
> `RedundantVersion: version 1.1.0: slot(0) keywords are overshadowed by version: 1.1.1`

Nah the RedundantVersion check is not really important, we skip it in the GitHub actions because it gets triggered every time there is more then one version with identical keywords.

Reply:
I changed the SRC_URI to what you suggested in the commit https://github.com/gentoo/sci/pull/1100/commits/53850a631f8c0ea277b97c6f492c3f913fad68ef